client/tracks/index.js

```diff
@@ -95,6 +95,8 @@ const events = {
 		'wcpay_subscriptions_account_not_connected_product_modal_finish_setup',
 	SUBSCRIPTIONS_ACCOUNT_NOT_CONNECTED_PRODUCT_MODAL_DISMISS:
 		'wcpay_subscriptions_account_not_connected_product_modal_dismiss',
+	DISPUTE_RESOLVED_MESSAGE_VIEW_BUTTON_CLICK:
+		'wcpay_dispute_resolved_message_view_button_click',
```

Comment:
This tracks event is better named 😁 

~~Though I still wonder if the user action here is different things (which we'd need to dig into props to discover). What are the actions here – `view_dispute_details_click`~~

OK that confirms how confused I was! 🤣 These buttons all do one action – viewing the dispute details (including submitted evidence etc). So the button function (and the user action / tracks event) is something like `wcpay_payment_details_view_dispute_details_click` or `wcpay_payment_details_view_dispute_submission_click` – is that accurate or do I misunderstand?

Reply:
Renamed again to `wcpay_payment_details_view_dispute_evidence_button_click`

Which aims to represent "the merchant clicked the payment details _View dispute evidence details_ button".